### PR TITLE
feat(services) timeout policy

### PR DESCRIPTION
**Pull request by lobkovilya**
GUI part for https://github.com/kumahq/kuma/pull/1554

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -315,6 +315,15 @@ export default (store) => {
...
           component: () => import(/* webpackChunkName: "retries" */ '@/views/Policies/Retries')
+        },
+        // timeouts
+        {
+          path: 'timeouts',
+          name: 'timeouts',
+          meta: {
+            title: 'Timeouts'
+          },
+          component: () => import(/* webpackChunkName: "retries" */ '@/views/Policies/Timeouts')
```
Comment (tharun208) [inline src/router.js:326]: can you change the chunk name to `timeouts` and also add the file extension to it. it should be '@/views/Policies/Timeouts.vue'

**Review by tharun208 — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/menu.js [hunk 2]
```diff
@@ -126,6 +126,12 @@ export default [
               link: 'retries',
               title: false,
               parent: 'policies'
+            },
+            {
+              name: 'Timeouts',
+              link: 'timeouts',
+              title: false,
+              parent: 'timeouts'
```
Comment (tharun208) [inline src/components/Sidebar/menu.js:134]: parent -> `policies`

**Review by tharun208 — CHANGES_REQUESTED** [review]

File: src/router.js (hunk 1 above)
Comment (lobkovilya) [reply src/router.js]: Thanks, fixed!

File: src/components/Sidebar/menu.js (hunk 2 above)
Comment (lobkovilya) [reply src/components/Sidebar/menu.js]: Thanks, fixed!

**Review by jakubdyszkiewicz — APPROVED** [review]

**Pull request merged**